app/screens/create_direct_message/user_list.tsx

```diff
@@ -180,31 +161,27 @@ export default function UserList({
         }
 
         return (
-            <View style={style.loadingContainer}>
-                <FormattedText
-                    id='mobile.loading_members'
-                    defaultMessage='Loading Members...'
-                    style={style.loadingText}
+            <View style={[style.noResultContainer, {paddingBottom: keyboardHeight}]}>
```

Comment:
I don't like creating new styles on every render, specially since down in the Loading step we are calling a library component (Lottie), but we can leave it to solve in the future if it is a real problem. (Also, probably changes on Loading have to be made too)